### DISCUSSION TITLE
efficient net 4 and resnet 50

### DISCUSSION THREAD

**Discussion by georgeper2001** [azavea/raster-vision#2083] [General]
Is efficient net 4 or efficient net 0 for Chip Classification and resnet 50 and resnet 18 are used for Object Detection?
can you use efficient net 4 for Object?

**Comment by AdeelH** [comment]
For chip classification, the default model is a ResNet (see available sizes [here](https://pytorch.org/vision/main/models/resnet.html)). An example of using an EfficientNet can be seen [here](https://github.com/azavea/raster-vision/blob/master/rastervision_pytorch_backend/rastervision/pytorch_backend/examples/chip_classification/spacenet_rio.py#L149-L159).

For object detection, the default model is a [FasterRCNN](https://pytorch.org/vision/main/models/faster_rcnn.html). Using an external model is trickier; see example [here](https://github.com/azavea/raster-vision/blob/master/rastervision_pytorch_backend/rastervision/pytorch_backend/examples/object_detection/cowc_potsdam.py#L144-L181).
